### DISCUSSION TITLE
GitHub Community Content

### DISCUSSION THREAD

**Discussion by MSTechMedia** [simatic-ax/.discussions#7] [Q&A] [answered]
What **type of content** would you like to have in our GitHub Community?

E.g.:
Examples
Deep Dives
Libraries
etc.

**Comment by BeckerStS** [comment, marked as answer]
Hi @MSTechMedia 

For now we would like to focus on the following topics:

[Application Examples] for SIMATIC AX. [overview](https://github.com/simatic-ax/.github/blob/main/docs/overview/appl-example.md)
[Code Snippets] that can be reused in AX Code. [overview](https://github.com/simatic-ax/.github/blob/main/docs/overview/code-snippets.md)
[Example Libraries] for usage in your own projects and idea creation. [overview](https://github.com/simatic-ax/.github/blob/main/docs/overview/example-libraries.md)
[Project Templates] for a head start in content creation. [overview](https://github.com/simatic-ax/.github/blob/main/docs/overview/templates.md)
[Tutorials] for self-study. [overview](https://github.com/simatic-ax/.github/blob/main/docs/overview/tutorials.md)

These are the most important topics that helps the community/our customers to develop simatic-libraries and simatic-applications more efficient. Time will tell how the community and the product is evolving, which may or may not leaves it open for new content types.

Cheers
